com.unity.render-pipelines.high-definition/Runtime/Material/SubsurfaceScattering/SubsurfaceScattering.compute

```diff
@@ -374,8 +376,8 @@ void SubsurfaceScattering(uint3 groupId          : SV_GroupID,
     // Reconstruct the view-space position corresponding to the central sample.
     float2 centerPosNDC = posInput.positionNDC;
     float2 cornerPosNDC = centerPosNDC + 0.5 * _ScreenSize.zw;
-    float3 centerPosVS  = ComputeViewSpacePosition(centerPosNDC, centerDepth, UNITY_MATRIX_I_P);
```

Comment:
If i understand correctly you are decompressing the oblique depth with the non-oblique matrix right? If yes this is incorrect

Reply:
@adrien-de-tocqueville

Reply:
yes that was indeed incorrect, I think the actual fix is simpler than that